islandora_iiif_manifests.module

```diff
@@ -19,6 +19,19 @@ function islandora_iiif_manifests_menu() {
       'file' => 'includes/manifest.inc',
       'type' => MENU_CALLBACK,
       'access callback' => TRUE,
+    ), 'iiif_manifest/%islandora_object/collection/%' => array(
```

Comment:
What is the exact meaning of this url? The argument after collection is the newspaper issue year?

Reply:
Yes, I have changed this to to volume. Because in volumes you can have several issues.

Reply:
Strictly speaking, it is not (always) a volume, but a year. If you want a more broader term, you could use "part". Library people are very strict about these terms...